config/prepare.js

```diff
@@ -1,3 +1,39 @@
+const redColor = '\x1b[31m';
+const greenColor = '\x1b[32m';
+const whiteColor = '\x1b[37m';
+const yellowColor = '\x1b[33m';
+
+// Verify Puppeteer configuration
+const pupSkip = process.env.PUPPETEER_SKIP_CHROMIUM_DOWNLOAD;
+
+if (!pupSkip) {
+    console.log(yellowColor, `Warning PUPPETEER_SKIP_CHROMIUM_DOWNLOAD is undefined,
+the installation'll be longer because Puppeteer'll download Chromium,
+only needed for testing. Read CODING.md for more information.`);
+}
+
+// Check node version to prevent error in THREE examples installation
+const nodeMajorVersion = process.versions.node.split('.')[0];
+const minNodeMajorVersion = 10;
+
+if (nodeMajorVersion < minNodeMajorVersion) {
+    console.log(redColor);
+    console.log('Node.js version :', process.versions.node);
+    console.error(`Error:\tYour installed Node version is inferior to ${minNodeMajorVersion},
+\tyou must have Node.js ${minNodeMajorVersion} or superior to develop in iTowns core`);
+} else {
+    console.log(greenColor, 'Node.js version :', process.versions.node);
+}
+
+const regex = new RegExp(/npm\/([^ ]+)/g);
+const npmVersion = regex.exec(process.env.npm_config_user_agent)[1];
+
+if (npmVersion) {
+    console.log(greenColor, 'Npm version :', npmVersion, '\n');
+}
+
+console.log(whiteColor);
```

Comment:
Did you actually meant to reset to the default? (what about someone using a white background?) ⇒ `\033[0m;`?

Fwiw, using chalk would remove that need, as it automatically resets after each message.

Reply:
I spotted `chalk`, but I couldn't use it in the **`presinstall`** script because it was not installed yet. But it's different now. I'm going to change that.